sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/implementation/util/DefaultLogger.java

```diff
@@ -452,20 +450,6 @@ private static String getFormattedDate() {
         return new String(bytes, StandardCharsets.UTF_8);
     }
 
-    private String addExceptionStacktrace(String message, Throwable t) {
-        if (t != null) {
-            message += "\",\"exception.stacktrace\":\"" + getStackTrace(t) + "\"";
```

Comment:
Is this dropping the stack trace logging or just handling it elsewhere?

Reply:
it adds the context props in ClientLogger - 
https://github.com/Azure/azure-sdk-for-java/blob/919e3da1f1bfdd0143db7441757ceccbae9d43f7/sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/util/ClientLogger.java#L490-L496

needed to simplify life, but also reliably add closing `}`. Arguably it's a bit less efficient, but if exception is throw it's already so bad for perf that resizing context list is super-minor.